.idea/compiler.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@PiLiGRiM18 I have big doubts regarding adding .idea files to the project. It should not be specifically limited to be used by any IDE. Could you please
1. remove this file from commit
2. if this file is necessary add steps to the description on how to create it.